src/ui/promgr/pmsdriver/local/local.go

```diff
@@ -131,14 +131,32 @@ func (d *driver) Update(projectIDOrName interface{},
 // List returns a project list according to the query parameters
 func (d *driver) List(query *models.ProjectQueryParam) (
 	*models.ProjectQueryResult, error) {
-	total, err := dao.GetTotalOfProjects(query)
-	if err != nil {
-		return nil, err
-	}
-
-	projects, err := dao.GetProjects(query)
-	if err != nil {
-		return nil, err
+	var total int64
+	var projects []*models.Project
+	var err error
+
+	if query != nil && query.User != nil && len(query.User.GroupList) > 0 {
```

Comment:
How can we handle the request that wants all project a user is a member of(no matter user member or group member)?